client/signup/main.jsx

```diff
@@ -626,7 +626,15 @@ class Signup extends React.Component {
 		if ( this.props.flowName !== 'onboarding-dev' ) {
 			return false;
 		}
-		const stepsToShowOn = [ 'site-topic', 'about', 'site-information', 'domains' ];
+		const stepsToShowOn = [
+			'site-topic',
+			'about',
+			'site-information-title',
+			'site-information-address',
+			'site-information-phone',
+			'domains',
+			'plans',
+		];
```

Comment:
This array will be redundant if we relocate the preview config to the step definitions, as #29400 attempts to do